pkg/feature/feature.go

```diff
@@ -108,22 +100,19 @@ func (f *Feature) Apply() error {
 
 func (f *Feature) Cleanup() error {
 	if !f.Enabled {
-		f.Log.Info("feature is disabled, skipping")
-
 		return nil
 	}
 
+	// Ensure associated FeatureTracker instance has been removed as last one
+	// in the chain of cleanups.
+	f.cleanups = append(f.cleanups, removeFeatureTracker)
```

Comment:
why not do 
f.addCleanup(removeFeatureTracker)

Reply:
it was late... :) thank you! changed in 481f10f